SCMS.Services.Api/Brokers/Storages/StorageBroker.AdditionalDetail.cs

```diff
@@ -0,0 +1,14 @@
+﻿// -----------------------------------------------------------------------
```

Comment:
We need relationships established to User table for CreatedBy and UpdatedBy, then to Student table for Student Id.

Please refer the video here - https://youtu.be/xrtFChDcU8I